carts/views.py

```diff
@@ -1,3 +1,36 @@
-from django.shortcuts import render
+import json
+
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models          import Cart
+from core.utils       import signin_decorator
 
 # Create your views here.
+
+class CartView(View):
+    @signin_decorator
+    def post(self, request):
+        try:
+            data       = json.loads(request.body)
+            user       = request.user
+            product_id = data['product_option_id']
```

Comment:
의미가 혼동될 수 있으니 변수명을 맞춰주면 좋을 것 같습니다.
```suggestion
            product_option_id = data['product_option_id']
```

Reply:
넵 수정하겠습니다!